src/DisplayReviews.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import Review from "./Review";
+
+const DisplayReviews = () => {
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    axios
```

Comment:
Nice use of loading state here but could we extract the api call in a different function? :)